sample/src/main/java/com/robotemi/sdk/sample/MainActivity.java

```diff
@@ -0,0 +1,313 @@
+                hideKeyboard(MainActivity.this);
+            }
+        }
+    }
+
+    /*
+        Simple follow me example.
+     */
+
+    public void followMe(View view) {
+        robot.beWithMe();
+        hideKeyboard(MainActivity.this);
+    }
+
+    /*
+        Manually navigate the robot with skidJoy, tiltAngle, turnBy and tiltBy.
+     */
+
+    public void skidJoy(View view) {
+        robot.skidJoy(1.0F, 1.0F);
+    }
+
+    public void tiltAngle(View view) {
+        robot.tiltAngle(23, 5.3F);
+    }
+
+    public void turnBy(View view) {
+        robot.turnBy(90, 6.2F);
+    }
+
+    public void tiltBy(View view) {
+        robot.tiltBy(30, 1.2F);
+    }
+
+    /*
+        Hiding keyboard after every button press
+     */
+
+    public static void hideKeyboard(Activity activity) {
+        InputMethodManager imm = (InputMethodManager) activity.getSystemService(Activity.INPUT_METHOD_SERVICE);
+        //Find the currently focused view, so we can grab the correct window token from it.
+        View view = activity.getCurrentFocus();
+        //If no view currently has focus, create a new one, just so we can grab a window token from it
+        if (view == null) {
+            view = new View(activity);
+        }
+        imm.hideSoftInputFromWindow(view.getWindowToken(), 0);
+    }
+
+    /*
+        Display the saved locations in a dialog
+     */
+
+    public void savedLocationsDialog(View view) {
+        hideKeyboard(MainActivity.this);
+        locations = robot.getLocations();
+        CustomAdapter customAdapter = new CustomAdapter(MainActivity.this, android.R.layout.simple_selectable_list_item, locations);
+        AlertDialog.Builder versionsDialog = new AlertDialog.Builder(MainActivity.this);
+        versionsDialog.setTitle("Saved Locations:");
+        versionsDialog.setPositiveButton("OK", null);
+        versionsDialog.setAdapter(customAdapter, null);
+        AlertDialog dialog = versionsDialog.create();
+        dialog.show();
+    }
+
+
+    // TODO: 2019-06-16 Work on onNlpCompleted. At the moment I couldn't get it to load.
+    @Override
+    public void onNlpCompleted(NlpResult nlpResult) {
+        //do something with nlp result. Base the action on what you get back
+        Toast.makeText(MainActivity.this, nlpResult.action, Toast.LENGTH_SHORT).show();
+    }
+
+    @Override
+    public void onWakeupWord(String wakeupWord) {
+       // Do anything on wakeup. Follow, go to location, or even try creating dance moves.
+    }
+
+    @Override
+    public void onTtsStatusChanged(TtsRequest ttsRequest) {
+
+        // Do whatever you like upon the status changing. after the robot finishes speaking
+        // Toast.makeText(this, "speech: " + ttsRequest.getSpeech() + "\nstatus:" + ttsRequest.getStatus(), Toast.LENGTH_LONG).show();
+    }
+
+    @Override
+    public void onBeWithMeStatusChanged(String status) {
+      //  When status changes to "lock" you can snap a picture.
```

Comment:
this comment is for Kellogs write something about  it means the robot recognize a user and following after him